docs/faq.rst

```diff
@@ -122,6 +123,10 @@ really desperate (e.g. if you have no completed backup of that file and you'ld
 rather get a partial file extracted than nothing). You do **not** want to give
 that option under any normal circumstances.
 
+Note that checkpoints inside files are created only since version 1.1, 
```

Comment:
Not sure if we want to include this into the faq. There are quite some features added or changing between versions and our changelog lists them. There is also a shorter changelog linked from the website.

Reply:
I think it would be good to have it in FAQ, at least as long as there are outdated versions shipped with Debian etc. - if others have the same problem as I had, probably it's much faster than looking into changelogs.

Reply:
ok, let's add it to faq.

2 wishes:
- please remove the empty line you accidentally added.
- can you also do a backport of this change to 1.1-maint branch (that's where 1.1.7 docs will come from, master will get into 1.2, but that might still take a while).

thanks!